core/domain/question_domain.py

```diff
@@ -1095,6 +1095,30 @@ def _convert_state_v44_dict_to_v45_dict(cls, question_state_dict):
 
         return question_state_dict
 
+    @classmethod
+    def _convert_state_v45_dict_to_v46_dict(cls, question_state_dict):
+        """Converts from version 45 to 46. Version 46 adds a new
+        customization arg to NumericInput interaction which allows
+        creators to set input range greater than or equal to zero.
+
+        Args:
+            question_state_dict: dict. A dict where each key-value pair
+                represents respectively, a state name and a dict used to
+                initialize a State domain object.
+
+        Returns:
+            dict. The converted question_state_dict.
+        """
+        if question_state_dict['interaction']['id'] == 'NumericInput':
+            customization_args = question_state_dict[
+                'interaction']['customization_args']
+            customization_args.update({
+                'placeholder': {
+                    'value': True
+                }
+            })
```

Comment:
Should the default be allow/disallow -ve? Do we need to audit the existing content on the server to decide how to handle this?

Reply:
corrected